### DISCUSSION TITLE
Post "https://accounts.google.com/o/oauth2/token": EOF

### DISCUSSION THREAD

**Discussion by CX-ZG** [casdoor/casdoor#2680] [General]
When I tried to log in using Google, I encountered the following error 

Post "https://accounts.google.com/o/oauth2/token": EOF

**Comment by hsluoyz** [comment]
See docs: https://casdoor.org/docs/basic/server-installation#:~:text=false%0AauthState%20%3D%20%22casdoor%22-,socks5Proxy,-%3D%20%22127.0.0.1%3A10808%22

**Reply by CX-ZG** [reply]
I don't think it's a proxy issue because if I shut down my proxy service, an error message will appear as Post“ https://accounts.google.com/o/oauth2/token ": socks connect TCP 10.0.8.11:7892->accounts. Google. com: 443: dial TCP 10.0.8.11:7892: connect: connection reused"

Here is my app. conf:

Appname=casdoor

HTTP port=8080

Runmode=dev

SessionOn=true

Copyrequestbody=true

DriverName=MySQL

DataSource Name=casdoor: F2mhaimZiLdzCYJt@tcp (10.0.8.11:3306)/casdoor

#DbName=casdoor

TableNamePrefix=

ShowSql=false

RedisEndpoint=

DefaultStorageProvider=

IsCloudIntranet=false

AuthState="casdoor"

Socks5Proxy="10.0.8.11:7892"

VerificationCodeTimeout=10

InitScore=2000

LogPostOnly=false

Origin=“ https://auth.chenchen.site “

StaticBaseUrl=“ https://cdn.casbin.org “

EnableGzip=true

**Reply by hsluoyz** [reply]
@CX-ZG try demo site: https://door.casdoor.com/login , it also uses the proxy and doesn't have any issue

![image](https://github.com/casdoor/casdoor/assets/3787410/4947ed3d-7120-435b-8b9a-49214999f34b)
